### PR TITLE
WIP - Orders panel part 3 (Add wallet + orders to the sidebar)

### PR DESCRIPTION
**Pull request by biocom**
# Summary

This brings the account details component inside the orders sidebar component:

https://user-images.githubusercontent.com/31534717/128858248-9ccd826c-fb41-46bf-8923-6a74672e1fcd.mov

A couple of notes:
- The wallet modal is opened with `toggleWalletModal` without setting the openOptions (what we currently do). When this prop was being passed, there was no toggling happening for me (wallet modal didn't open). Feel free to iterate in this PR if you can make it work. Right now it does open the wallet modal, but not in the `WALLET_VIEWS.ACCOUNT` state.
- If you close the Wallet modal (when you hit the 'change' button in the account details), it also closes the sidebar. This is because we have `useOnClickOutside` event in place (close the orders sidebar if you click outside of it). We could iterate in a future PR on this.

### CODE REVIEW

File: src/components/WalletModal/index.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ import usePrevious from '../../hooks/usePrevious'
 import { ApplicationModal } from '../../state/application/actions'
 import { useModalOpen, useWalletModalToggle } from '../../state/application/hooks'
 import { ExternalLink, TYPE } from 'theme'
-import AccountDetails from 'components/AccountDetails'
+import AccountDetails from '@src/components/AccountDetails'
```
Comment (biocom) [inline src/components/WalletModal/index.tsx:18]: Thanks @anxolin now seeing you do it, makes sense. As the original UNI file was pulling our mod file.

**Comment by W3stside** [comment]
@biocom I'm sure you're aware but the build is broken

**Comment by biocom** [comment]
> @biocom I'm sure you're aware but the build is broken

Apparently Mac has case (in)sensitive settings on. Locally my folder was `OrdersPanel` but it did not commit this. The solve was to do a round of:
`git mv old temp`
`git mv temp new`

Then commit. Just FYI.

File: src/custom/components/AccountDetails/index.tsx [hunk 2]
```diff
@@ -24,10 +24,8 @@ import { clearOrders } from 'state/orders/actions'
 import {
   WalletName,
   MainWalletAction,
-  AccountDetailsProps,
+  // AccountDetailsProps,
```
Comment (alfetopito) [inline src/custom/components/AccountDetails/index.tsx:27]: Not a mod, you can delete this

**Review by alfetopito — APPROVED** [review]
Only 2 minor comments.

**Comment by elena-zh** [comment]
Hey @biocom , great changes!
Some issues: 
1. There is still no 'Recent activity' name in the dark mode (reported in #1113)
recent

2. It would be nice to remove a boarder radius at the bottom of the 'recent activity' area
boarder radius

3. 'Connected with Unknown wallet' is displayed in the Side bar when disconnect a wallet
unknown wallet

**Comment by W3stside** [comment]
@biocom should clicking outside the wallet modal close the activities modal too? i think it would be kind of annoying imo

**Comment by biocom** [comment]
> @biocom should clicking outside the wallet modal close the activities modal too? i think it would be kind of annoying imo

It should correct. My argument is that this is a standard 'expected' UX pattern.

**Comment by W3stside** [comment]
> > @biocom should clicking outside the wallet modal close the activities modal too? i think it would be kind of annoying imo
> 
> It should, correct. My argument is that this is a standard 'expected' UX pattern.

the 2nd modal i mean, the activities modal outside click closing is expected. but the second modal that pops up closes both on clicking outside. anyways if it's annoying later on from users we can change it i will approve.

**Review by W3stside — APPROVED** [review]

**Comment by biocom** [comment]
For the second modal, we should address this in a later PR. I agree that it can be annoying.

**Comment by alfetopito** [comment]
> > @biocom should clicking outside the wallet modal close the activities modal too? i think it would be kind of annoying imo
> 
> It should, correct. My argument is that this is a standard 'expected' UX pattern.

I agree with Michel and also prefer that option.

**Pull request merged**